### PR TITLE
Sound fix

### PR DESCRIPTION
**Pull request by OnlyBMan**
This should fix the /playsound and /stopsound commands on MCPE.

### CODE REVIEW

File: src/protocolsupport/protocol/packet/middle/clientbound/play/MiddleWorldCustomSound.java [hunk 1]
```diff
@@ -23,7 +24,15 @@ public MiddleWorldCustomSound(ConnectionImpl connection) {
 
 	@Override
 	public void readFromServerData(ByteBuf serverdata) {
-		id = StringSerializer.readString(serverdata, ProtocolVersionsHelper.LATEST_PC);
+		if(connection.getVersion() == ProtocolVersionsHelper.LATEST_PE)
```
Comment (colinrgodsey) [inline src/protocolsupport/protocol/packet/middle/clientbound/play/MiddleWorldCustomSound.java]: can do *connection.getVersion().isPE()* here, keeps future-proofing. can add conditionals for version specifics later if/when they come up

Comment (colinrgodsey) [reply src/protocolsupport/protocol/packet/middle/clientbound/play/MiddleWorldCustomSound.java]: also, you'll want to modify the PE extended version of this class (protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldCustomSound). The middle packets themselves shouldnt have anything version specific- they're basically the interface layer for the native server version.

File: src/protocolsupport/protocol/packet/middle/serverbound/play/MiddleStopSound.java [hunk 2]
```diff
@@ -13,6 +13,8 @@ public MiddleStopSound(ConnectionImpl connection) {
 
 	//TODO: structure
 	protected ByteBuf data;
+	protected String name = "";
```
Comment (colinrgodsey) [inline src/protocolsupport/protocol/packet/middle/serverbound/play/MiddleStopSound.java]: ~~same here, but this class actually doesnt have a PE subclass yet- so you'll need to add it (if PE has an equivalent)~~ didnt look down more lol but same, shouldnt need to modify the middle packets unless they're missing some definition/parsing from the native/1.13 packets

File: src/protocolsupport/protocol/pipeline/version/v_pe/PEPacketEncoder.java [hunk 3]
```diff
@@ -9,66 +9,7 @@
...
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.StartGame;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.TimeUpdate;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.Title;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.UnloadChunk;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.VehiclePassengers;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldCustomSound;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldEvent;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldParticle;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldSound;
+import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.*;
```
Comment (colinrgodsey) [inline src/protocolsupport/protocol/pipeline/version/v_pe/PEPacketEncoder.java]: intellij is evil like this >.< it sucks, but should try to keep imports explicit

File: src/protocolsupport/protocol/pipeline/version/v_pe/PEPacketEncoder.java [hunk 4]
```diff
@@ -132,6 +73,7 @@
 		registry.register(NetworkState.PLAY, ClientBoundPacket.PLAY_EXPLOSION_ID, Explosion::new);
 		registry.register(NetworkState.PLAY, ClientBoundPacket.PLAY_WORLD_SOUND_ID, WorldSound::new);
 		registry.register(NetworkState.PLAY, ClientBoundPacket.PLAY_WORLD_CUSTOM_SOUND_ID, WorldCustomSound::new);
+		registry.register(NetworkState.PLAY, ClientBoundPacket.PLAY_STOP_SOUND, StopSound::new);
```
Comment (colinrgodsey) [inline src/protocolsupport/protocol/pipeline/version/v_pe/PEPacketEncoder.java:136]: 👍 you found the client-bound packet registry

**Review by colinrgodsey — CHANGES_REQUESTED** [review]
some minor fixes, no worries. glad to see you submit this PR! hope to see more :)

**Comment by OnlyBMan** [comment]
My new commit should’ve fixed the issues!

**Pull request merged**